### PR TITLE
Move document variables into single yml file

### PR DESCRIPTION
**Pull request by stefanv**
The variable renames in the *content* files are already in my other PR.

### CODE REVIEW

File: scripts/set_version.py [hunk 1]
```diff
@@ -3,9 +3,27 @@
 """
 
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
+import yaml
+
 
 QUARTO_TEMPLATE = '_quarto.yml.template'
-VARS_TEMPLATE = '_variables.yml.template'
+
+LANG_VARS_SOURCE = 'variables.yml'
```
Comment (matthew-brett) [inline scripts/set_version.py:11]: Confusing name?  Maybe `input_variables.yml`?

File: scripts/set_version.py [hunk 2]
```diff
@@ -15,50 +33,25 @@ def get_parser():
     return parser
 
 
-def set_version(version):
+def fill_template(fn, version):
```
Comment (matthew-brett) [inline scripts/set_version.py:36]:
```suggestion
def fill_quarto_template(fn, version):
```

File: scripts/set_version.py [hunk 3]
```diff
@@ -15,50 +33,25 @@ def get_parser():
...
+    out_fname = fn.replace('.template', '')
+    with open(out_fname, 'wt') as fobj:
+        fobj.write(fmt_str.format(**variables))
 
 
 def main():
     parser = get_parser()
     args = parser.parse_args()
-    set_version(args.version)
+    fill_template(QUARTO_TEMPLATE, args.version)
```
Comment (matthew-brett) [inline scripts/set_version.py:50]:
```suggestion
    fill_quarto_template(QUARTO_TEMPLATE, args.version)
```

File: scripts/set_version.py [hunk 4]
```diff
@@ -15,50 +33,25 @@ def get_parser():
...
+
+    vars = yaml.load(open(LANG_VARS_SOURCE, 'r'), Loader=yaml.FullLoader)
```
Comment (matthew-brett) [inline scripts/set_version.py:52]: Not `SafeLoader`?

**Review by matthew-brett — COMMENTED** [review]
Usual nitty stuff.

File: scripts/set_version.py (hunk 1 above)
Comment (stefanv) [reply scripts/set_version.py:11]: Renamed it to `text_variables.yml`.

File: scripts/set_version.py (hunk 2 above)
Comment (stefanv) [reply scripts/set_version.py:36]: This is a generic filler, called as `    fill_template(QUARTO_TEMPLATE, args.version)`, so think it's OK as-is.

File: scripts/set_version.py (hunk 4 above)
Comment (stefanv) [reply scripts/set_version.py:52]: We specify the content, so no need for safe loading.

**Pull request merged**